src/api/forms/form-metadata-repository.js

```diff
@@ -0,0 +1,51 @@
+import * as fs from 'fs/promises'
+import { join } from 'path'
```

Comment:
Not sure if ESLint says this but we have named exports on `fs/promises`

Might have been a TypeScript/Babel compatibility thing at one point?

Good to prefix Node.js core packages with the `node:` prefix so they never clash with a registry. I think from Node.js 18 some packages are _only_ available with the prefix 😮

```suggestion
import { join } from 'node:path'
import { readdir, readFile } from 'node:fs/promises'
```